summit/summitxl_params.env

```diff
@@ -8,7 +18,7 @@ export ROBOT_XACRO_PACKAGE=waterloo_steel_description # waterloo steel simulatio
 # 24V motors: 12.52, 48V motors: 9.56
 export ROBOT_GEARBOX=9.56
 # skid, omni, steel_skid, steel_omni
-export ROBOT_KINEMATICS=steel_omni
+export ROBOT_KINEMATICS=omni
```

Comment:
Need to check out later once a sample demo is executed if there is any issue caused by the change (there might be a reason that Robotnik split the kinematic models into omni and steel_omni).

Reply:
To me, I think its some internal fancy words to differentiate the launch, like https://github.com/RobotnikAutomation/rbeken_common/blob/6d31e1950a54305f3a326d675eacefcb6adb1718/rbeken_navigation/launch/navigation_complete.launch#L5